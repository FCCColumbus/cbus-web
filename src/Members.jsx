

function Members() {
    const admin = [
        { name: 'John', imgUrl: '/head.png' },
        { name: 'Amy', imgUrl: '/head.png' },
        { name: 'Kevin', imgUrl: '/head.png' },
    ]
    const members = [
        { name: 'John', imgUrl: '/head.png' },
        { name: 'Ismael', imgUrl: '/head.png' },
        { name: 'Nirvignesh', imgUrl: '/head.png' },
        { name: 'Tijana', imgUrl: '/head.png' },
        { name: 'Jennifer', imgUrl: '/head.png' },
        { name: 'Kellan', imgUrl: '/head.png' },
    ]
    return (
        <>
            <div className="members">
                <h3>Meet the Admins</h3>
                <div className="row">
                    {admin.map((member) =>
                        <div key={member.name}>
                            <img className='admin' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />
                            <h4>{member.name}</h4>
                        </div>
                    )}
                </div>
            </div>
            <div className="members">
                <h3>Meet the Contributors</h3>
                <div className="row">
                    {members.map((member) =>
                        <div key={member.name}>
                            <img className='member' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />
                            <h4>{member.name}</h4>
                        </div>
                    )}
                </div>
            </div>
        </>
    )

}

export default Members