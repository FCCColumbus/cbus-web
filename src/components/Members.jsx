function Members() {
  const admin = [
    { id: 0, firstName: 'John', lastName: 'Riley', title: 'Admin & Coding Mentor', imgUrl: 'https://avatars.githubusercontent.com/u/41979303?v=4' },
    { id: 1, firstName: 'Kent', lastName: 'Utterback', title: 'Admin Emeritus & Coding Mentor', imgUrl: 'https://avatars.githubusercontent.com/u/17324971?v=4' },
    { id: 2, firstName: 'Amy', lastName: 'Bedinghaus', title: 'Admin Emeritus', imgUrl: 'https://avatars.githubusercontent.com/u/28076677?v=4' },
    { id: 3, firstName: 'Austin', lastName: 'Truss', title: 'Admin Emeritus', imgUrl: 'https://media.licdn.com/dms/image/C5603AQGQpynBhWiZLA/profile-displayphoto-shrink_200_200/0/1517355585710?e=1696464000&v=beta&t=DrPEzm5U1we4G7qUPoFrAA5gw_MHSN3dU8_02iWZYRA' },
    { id: 4, firstName: 'Kevin', lastName: 'Bruland', title: 'Admin Emeritus', imgUrl: 'https://avatars.githubusercontent.com/u/17850505?v=4' },
    { id: 5, firstName: 'Sarah', lastName: 'Lilly-Bruland', title: 'Admin Emeritus', imgUrl: 'https://avatars.githubusercontent.com/u/17872422?v=4' },
    { id: 6, firstName: 'Justin', lastName: 'Woodward', title: 'Admin Emeritus', imgUrl: 'https://avatars.githubusercontent.com/u/17994108?v=4' },
  ]

  return (
    <div>
      <div className="admins">
        <h3>Meet the Admins</h3>
        <div className="row">
          {admin.map((member) =>
            <div key={member.id}>
              <h4>{member.firstName}<br/> {member.lastName}</h4>
              <img className='avatar admin-avatar' src={member.imgUrl} alt={`beautiful headshot of ${member.firstName} ${member.lastName}.`} />
              <h5>{member.title}</h5>
            </div>
          )}
        </div>
      </div>
      <div className="contributors"> 
        <h3>Meet the Contributors</h3>
          <a className="contributors-link" href="https://github.com/FCCColumbus/cbus-web/graphs/contributors" rel="noreferrer" target="_blank">
            <img className='contributors-avatar' src='https://contrib.rocks/image?repo=FCCColumbus/cbus-web' alt='beautiful headshots of our members' />         
          </a>
      </div>
    </div>
  );
}

export default Members