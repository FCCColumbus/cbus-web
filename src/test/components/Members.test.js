import renderer from "react-test-renderer";

import Members from "../../Members";

describe("Members", () => {
    it("renders", () => {
        const view = renderer.create(<Members />);
        expect(view).toMatchSnapshot();
    });
});
