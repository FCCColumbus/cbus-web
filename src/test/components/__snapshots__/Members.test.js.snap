// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Members renders 1`] = `
<div>
  <div
    className="members"
  >
    <h3>
      Meet the Admins
    </h3>
    <div
      className="row"
    >
      <div>
        <h4>
          John
          <br />
           
          Riley
        </h4>
        <img
          alt="beautiful headshot of John Riley."
          className="avatar admin-avatar"
          src="https://avatars.githubusercontent.com/u/41979303?v=4"
        />
        <h5>
          Admin & Coding Mentor
        </h5>
      </div>
      <div>
        <h4>
          Kent
          <br />
           
          Utterback
        </h4>
        <img
          alt="beautiful headshot of Kent Utterback."
          className="avatar admin-avatar"
          src="https://avatars.githubusercontent.com/u/17324971?v=4"
        />
        <h5>
          Admin Emeritus & Coding Mentor
        </h5>
      </div>
      <div>
        <h4>
          Amy
          <br />
           
          Bedinghaus
        </h4>
        <img
          alt="beautiful headshot of Amy Bedinghaus."
          className="avatar admin-avatar"
          src="https://avatars.githubusercontent.com/u/28076677?v=4"
        />
        <h5>
          Admin Emeritus
        </h5>
      </div>
      <div>
        <h4>
          Austin
          <br />
           
          Truss
        </h4>
        <img
          alt="beautiful headshot of Austin Truss."
          className="avatar admin-avatar"
          src="https://media.licdn.com/dms/image/C5603AQGQpynBhWiZLA/profile-displayphoto-shrink_200_200/0/1517355585710?e=1696464000&v=beta&t=DrPEzm5U1we4G7qUPoFrAA5gw_MHSN3dU8_02iWZYRA"
        />
        <h5>
          Admin Emeritus
        </h5>
      </div>
      <div>
        <h4>
          Kevin
          <br />
           
          Bruland
        </h4>
        <img
          alt="beautiful headshot of Kevin Bruland."
          className="avatar admin-avatar"
          src="https://avatars.githubusercontent.com/u/17850505?v=4"
        />
        <h5>
          Admin Emeritus
        </h5>
      </div>
      <div>
        <h4>
          Sarah
          <br />
           
          Lilly-Bruland
        </h4>
        <img
          alt="beautiful headshot of Sarah Lilly-Bruland."
          className="avatar admin-avatar"
          src="https://avatars.githubusercontent.com/u/17872422?v=4"
        />
        <h5>
          Admin Emeritus
        </h5>
      </div>
      <div>
        <h4>
          Justin
          <br />
           
          Woodward
        </h4>
        <img
          alt="beautiful headshot of Justin Woodward."
          className="avatar admin-avatar"
          src="https://avatars.githubusercontent.com/u/17994108?v=4"
        />
        <h5>
          Admin Emeritus
        </h5>
      </div>
    </div>
  </div>
  <div
    className="contributors"
  >
    <h3>
      Meet the Contributors
    </h3>
    <a
      className="contributors-link"
      href="https://github.com/FCCColumbus/cbus-web/graphs/contributors"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="beautiful headshots of our members"
        className="members-avatar"
        src="https://contrib.rocks/image?repo=FCCColumbus/cbus-web"
      />
    </a>
    <div
      className="row contributors-name-container"
    >
      <h4>
        Jed
      </h4>
      <h4>
        John
      </h4>
      <h4>
        Amy
      </h4>
      <h4>
        Mike
      </h4>
      <h4>
        Ismael
      </h4>
      <h4>
        Nash
      </h4>
      <h4>
        Tijana
      </h4>
      <h4>
        Jennifer
      </h4>
      <h4>
        Kellan
      </h4>
      <h4>
        FCCCAdmin
      </h4>
    </div>
  </div>
</div>
`;
